tests/kernel/timer/timer_behavior/pytest/test_timer.py

```diff
@@ -52,6 +52,29 @@ def do_analysys(test, stats, config, sys_clock_hw_cycles_per_sec):
     logger.info(f'real drift:     {time_diff * 1_000_000:.6f} us')
     logger.info('====================================================')
 
+    logger.info('RECORD: {'
```

Comment:
`logger.info` will not be printed by twister, if you use verbose less than 2,
see: https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/pylib/twister/twisterlib/harness.py#L344
```
        if handler.options.verbose > 1:
            command.extend([
                '--log-cli-level=DEBUG',
                '--log-cli-format=%(levelname)s: %(message)s'
            ])
```
and then in `_ourtput_reader` method tou do not have any line to parse:
```
        while proc.stdout.readable() and proc.poll() is None:
            line = proc.stdout.readline().decode().strip()
            if not line:
                continue
            logger.debug('PYTEST: %s', line)
            harness.parse_record(line)
```

Test it with `twister -v`, no metrics in twister.json

Reply:
Thank you for pointing at this: the 'external' test will have no output logged following the Twister verbose level.
I've added logging options for pytest plugin in the testcase.yaml explicitly.

Reply:
It is not good solution to add it in testcase.yaml. You must then remember to update pytest-args in every sample/testcase yaml when you want to use such a metrics.
I think that now there is no reason to remove `--log-cli-level=DEBUG`, when verbosity is lower in twister (Twister does not print it).
So better to have generic solution by adding:
```
                '--log-cli-level=DEBUG',
                '--log-cli-format=%(levelname)s: %(message)s'
```
to command always.
Another benefit from it would be to have all debug logs in the twister output when pytest fails (with twister `--inline-logs`)

Reply:
> It is not good solution to add it in testcase.yaml. .... So better to have generic solution by adding: ...
> to command always. Another benefit from it would be to have all debug logs in the twister output when pytest fails (with twister `--inline-logs`)

I agree, it is a general pytest harness problem: to have both outputs (from the test image and from the  pytest test) to pass through pytest plugin into Twister logs.